code/go/0chain.net/smartcontract/minersc/miner.go

```diff
@@ -133,23 +133,30 @@ func (msc *MinerSmartContract) AddMiner(t *transaction.Transaction,
 	if _, ok := allMap[newMiner.GetKey()]; !ok {
 		allMiners.Nodes = append(allMiners.Nodes, newMiner)
 
-		if err = updateMinersList(balances, allMiners); err != nil {
-			return "", common.NewErrorf("add_miner",
-				"saving all miners list: %v", err)
-		}
-		update = true
-	}
+		//if err = updateMinersList(balances, allMiners); err != nil {
```

Comment:
Why remove this block? We are only emitting events in smartcontracts, not changing how miners work.

Reply:
done